libpldmresponder/platform.hpp

```diff
@@ -54,6 +54,7 @@ class Handler : public CmdHandler
             pldm_pdr* repo, HostPDRHandler* hostPDRHandler,
             pldm::state_sensor::DbusToPLDMEvent* dbusToPLDMEventHandler,
             fru::Handler* fruHandler,
+            pldm_entity_association_tree* bmcEntityTree,
```

Comment:
this is redundant addition in this PR?

Reply:
No it is not , I had to add it because of the class dependency